api/v1alpha1/olsconfig_types.go

```diff
@@ -102,6 +102,8 @@ type OLSSpec struct {
 	// User data collection switches
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="User Data Collection"
 	UserDataCollection UserDataCollectionSpec `json:"userDataCollection,omitempty"`
+
```

Comment:
we can add an API marker to tell how UI display it  `+operator-sdk:csv:customresourcedefinitions:displayName=`

Reply:
and xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}